### PR TITLE
27 fe docker image

### PR DESCRIPTION
**Pull request by Kayra**
# Production Dockerfile

In the client directory run:

```bash
docker build -t curation-client .
docker run -p 80:80 curation-client
```

The client should be accessible via [http://localhost:80](http://localhost:80)

Test connection to api by commenting out the `client` and `volumes` parts of the docker-compose.yml file and starting the api with the usual `make start` in the root directory, then refreshing the client to see if the data has been populated.

# Development Dockerfile

In the root directory run:

```bash
docker compose up
```

The client should be accessible via [http://localhost:3000](http://localhost:3000)

---

Closes #37.

### CODE REVIEW

**Comment by RobbyCowell** [comment]
Code makes sense and looks good to me. I'll try running it locally later today.

File: client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM node:12.2.0-alpine as build
```
Comment (RobbyCowell) [inline client/Dockerfile:1]: Maybe try `node:16.5.0` I can compile locally using v15.8.0, so we might want to move off of the LTS versions.

If this doesn't work, I'll try to lock-down the Node Sass version to be compatible with LTS

File: client/package.json [hunk 2]
```diff
@@ -39,5 +39,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "chokidar": "^3.5.2"
```
Comment (RobbyCowell) [inline client/package.json:44]: What is this?

**Review by RobbyCowell — APPROVED** [review]

File: client/package.json (hunk 2 above)
Comment (Kayra) [reply client/package.json:44]: hot reload in docker

**Pull request merged**